Movie Search/Cells/MovieCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MovieCell.swift
+//  Movie Search
+//
+//  Created by Alexey Grabik on 15/04/2020.
+//
+
+import UIKit
+
+class MovieCell: UITableViewCell {
```

Comment:
I'd opt for a more descriptive name - currently this could be either a table view or collection view cell.